app/views/layouts/_center_div_with_listnav.html.haml

```diff
@@ -1,3 +1,4 @@
+- left_div_classname =  @def_searches.blank? && @my_searches.blank? && !controller.respond_to?(:display_tree) ? "no_filters":""
```

Comment:
```suggestion
- left_div_classname =  no_filters?  ? "no_filters" : ""
```

where the `no_filters?` method is defined in a helper

Reply:
@chessbyte `no_filters` is just a class name i am declaring to write css. it is stored in variable called `left_div_classname` and i am rendering this variable in div class names.

Reply:
Interesting , any advantage in the helper method?  helper method just have 
`@def_searches.blank? && @my_searches.blank? && !controller.respond_to?(:display_tree)`?, right?
i have to write css anyways by adding class name.

Reply:
I think it just adds clarity to what all these if statements mean.  Additionally, it is easy to write specs against a helper method and, if you want to test the view, then all you do is stub the `no_filters?` method instead of all the components of it